### DISCUSSION TITLE
Recommendation for number of worker processes

### DISCUSSION THREAD

**Discussion by AlfredSK** [misskey-dev/misskey#9177] [Q&A] [answered]
Hello,
the config file has a setting for the number of worker processes. The default is 1.

```
# Number of worker processes
clusterLimit: 1
```

I wonder if it makes sense to raise that value. Any idea what to check to find out the best value?

**Comment by ThatOneCalculator** [comment]
I do ~1/3 of the amount of CPU threads on the server. On my instance, I have 12 cores so I set the `clusterLimit` to 4

**Comment by syuilo** [comment, marked as answer]
Short answer:
Usually 1 is fine.

Long answer:
I believe that Misskey's performance bottleneck is the database, and that the Node process is rarely the bottleneck.
Therefore, I believe that increasing the number of processes in a cluster will only increase the memory usage of the server and provide little performance benefit.
